resource-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/resource/kafka/KafkaInitializer.java

```diff
@@ -120,23 +120,23 @@ Materializer materializer(final ActorSystem system) {
     // Serialiser for upstream test input
     static class RequestSerializer implements Serializer<JsonNode> {
         @Override
-        public byte[] serialize(final String s, final JsonNode userRequest) {
+        public byte[] serialize(final String s, final JsonNode resourceRequest) {
```

Comment:
```suggestion
        public byte[] serialise(final String s, final JsonNode resourceRequest) {
```
This will need to be changed wherever it's used

Reply:
cant rename method that we're overriding.  Java uses the american spelling

Reply:
I think I meant to get the class name but got distracted by the method name.